### ISSUE TITLE
Require users to set `python_version`

### ISSUE THREAD

**Issue by bfirsh** [replicate/cog#101]
This will stop models from breaking when we update our default.

**Comment by zeke** [comment]
To clarify, are you suggesting that `python_version` should be a required property in `cog.yaml`?

I think you also mentioned something about wanting to rename this property to `python`, right? If so, it might be good to roll up these two changes together.

**Comment by bfirsh** [comment]
Yes, that's right. Not sure if renaming is the right thing to do, but if it is, that sounds good.

Some of these issues are hastily written with a bunch of assumed knowledge -- sorry. Feel free to edit the main issue if they need clarification. Sometimes I treat the main issue a bit like a wiki, and add a note to the issue to say it was written by / updated by multiple people: https://github.com/replicate/cog/issues/29

**Comment by zeke** [comment]
> Feel free to edit the main issue if they need clarification

Updated the issue title. 👍🏼

**Comment by mcam10** [comment]
I think we can close this issue out?
